shared/functions.js

```diff
@@ -50,6 +50,22 @@ export function makeResponse (
   return {event, data}
 }
 
+export function makeLog (
+  groupId: string,
+  currentLogPosition: string
+): Log {
+  return {groupId, currentLogPosition}
+}
```

Comment:
Nevermind, I see that you're using the `Log` type in `UserSession`. Can you instead then add comments to both `Log` and `UserSession` to "delete these later if possible."